examples-next/images/keystone.ts

```diff
@@ -26,6 +26,16 @@ export default withAuth(
       url: process.env.DATABASE_URL || 'postgres://keystone5:k3yst0n3@localhost:5432/todo-example',
     },
     lists,
+    images: {
+      /* could be 'cloud' */
+      upload: 'local',
```

Comment:
@JedWatson I personally feel a much clearer and more correct key name here is `storage`. This is common in other frameworks (https://pypi.org/project/Flask-Storage/, https://laravel.com/docs/8.x/filesystem) and is easy for the user to understand as you are essentially configuring where your images are being stored. On from this, the options should be `'filesystem' | 'cloud'` because, again, these are the most clear to the user in terms of what is actually happening.